docs/checkmCmds.txt

```diff
@@ -1,6 +1,6 @@
 root@ad340d32091a:/kb/module# checkm
 
-                ...::: CheckM v1.0.7 :::...
+                ...::: CheckM v1.1.2 :::...
```

Comment:
Copied and pasted the relevant commands and their output from the new version of CheckM